integration/spark/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchema.scala

```diff
@@ -432,7 +440,9 @@ case class LoadTable(
       val dateFormat = options.getOrElse("dateformat", null)
       ValidateUtil.validateDateFormat(dateFormat, table, tableName)
       val maxColumns = options.getOrElse("maxcolumns", null)
-      val sortScope = options.getOrElse("sort_scope", null)
+
+      val tableProperties = table.getTableInfo.getFactTable.getTableProperties
+      val sortScope = if (null == tableProperties) null else tableProperties.get("sort_scope")
```

Comment:
If tableProperties is null, it need to use default sort scope, right?

Reply:
I use CarbonCommonConstants.LOAD_SORT_SCOPE_DEFAULT as its default value.